renderdocui/Windows/MainWindow.cs

```diff
@@ -1006,18 +1006,32 @@ private LiveCapture OnCaptureTrigger(string exe, string workingDir, string cmdLi
             if (!PromptCloseLog())
                 return null;
 
-            string logfile = m_Core.TempLogFilename(Path.GetFileNameWithoutExtension(exe));
-
-            UInt32 ret = m_Core.Renderer.ExecuteAndInject(exe, workingDir, cmdLine, env, logfile, opts);
+            UInt32 remoteIdent;
+            if (m_Core.Renderer.Remote != null && m_Core.Renderer.Remote.Hostname.StartsWith("adb:"))
```

Comment:
I think this is better placed to go in the remote server on the C++ side. We already know if we are connecting to android in `RENDERDOC_CreateRemoteServerConnection` which can be passed through creation, and the code can do something different inside `RemoteServer::ExecuteAndInject`.